modules/evm/src/lib.rs

```diff
@@ -1343,15 +1345,17 @@ impl<T: Config> Pallet<T> {
 	#[transactional]
 	pub fn remove_contract(caller: &EvmAddress, contract: &EvmAddress) -> DispatchResult {
 		let contract_account = T::AddressMapping::get_account_id(contract);
-
-		Accounts::<T>::try_mutate_exists(contract, |account_info| -> DispatchResult {
+		let mut task_id = Default::default();
+		Accounts::<T>::try_mutate_exists(contract, |maybe_account_info| -> DispatchResult {
```

Comment:
`try_mutate_exists` can return `task_id`, just change `DispatchResult` to `Result<Nonce, DispatchError>`